website/layouts/partials/page-meta-links.html

```diff
@@ -1,9 +1,9 @@
-{{ if .Path }}
+{{ if .File.Path }}
```

Comment:
This change is because in newer version of Hugo `.Path` is deprecated, and I saw an warning message.